tests/bluetooth/controller/mock_ctrl/src/ull_conn_iso.c

```diff
@@ -42,21 +42,23 @@
 #include "hal/debug.h"
 
 static struct ll_conn_iso_group cig = { 0 };
-static struct ll_conn_iso_stream cis = { .established = 1};
+static struct ll_conn_iso_stream cis = { 0 };
 
-struct ll_conn_iso_stream *ll_conn_iso_stream_get_by_acl(struct ll_conn *conn, uint16_t *cis_iter)
+__attribute__((weak))
```

Comment:
in include/zephyr/toolchain/gcc.h __weak is defined as __attribute__((weak)), and this is used for weak functions
```suggestion
__weak
```

Reply:
OK